src/_templates/main_loop_plugin.rst

```diff
@@ -7,6 +7,10 @@
       otherwise you can end up with imported modules and mess like that
 
    {% block functions %}
+
+   Python API
+   ----------
+
```

Comment:
This header separates the API docs in the main-loop plugins sections from the rest of the content as it wasn't really clear what they meant before.